### DISCUSSION TITLE
USB Ethernet Setup

### DISCUSSION THREAD

**Discussion by tdashmike** [gtxaspec/wz_mini_hacks#667] [Q&A]
Hello, I just had a few question as I'm about to move my cameras to a wired connection.

1. The micro usb to usb a cable it comes with has no data wires in it correct?
2. When I turn on USB Ethernet in the config, does it turn off wireless immediately?
3. If it doesn't turn off wireless, how do i turn it off later when the ethernet connection is working?
4. If it turns it off and I can't get my ethernet working, is the only way to recover to pull the sd card and turn off usb Ethernet?

Thank you!

**Comment by virmaior** [comment]
1. correct.
2. No, it has to turn on the wifi initially and then switch them out
3. It turns off the wifi automatically once it turns on the usb ethernet
4. There's a bonding mode that's supposed to help with that (I've never tried it).

**Reply by tdashmike** [reply]
Didn’t even know bonding was in the config. I will try it out and report back if it works.

The reason I was asking is because I have 2 cameras under the roof that are hard to get to.

**Reply by virmaior** [reply]
When wz_mini_hack just came out, I spent hours trying to do the impossible because I hadn't know the original cable has no data lines.

**Reply by tdashmike** [reply]
If it makes you feel any better, I threw away a device one time and later found out all the usb cables I tried with it had no data lines!
